server/src/main/java/org/elasticsearch/action/fieldcaps/FieldCapabilities.java

```diff
@@ -120,6 +130,9 @@ public void writeTo(StreamOutput out) throws IOException {
     public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
         builder.startObject();
         builder.field(TYPE_FIELD.getPreferredName(), type);
+        if (isMetadataField) {
+            builder.field(METADATA_FIELD.getPreferredName(), isMetadataField);
+        }
```

Comment:
thinking out loud: would it make sense to always print it? otherwise consumers need to account for the case where the field is not there, as well as for the case where the field is there and is false, which never happens.

Reply:
That makes sense. I pushed https://github.com/elastic/elasticsearch/pull/69977/commits/78fb83ed12c190e52aa24eca0cd8b814271f5bee